jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -810,7 +810,13 @@ function configure_server_state() {
         # create a KIE server state file with all configured containers and properties
         local stateFileInit="org.kie.server.services.impl.storage.file.KieServerStateFileInit"
         log_info "Attempting to generate kie server state file with 'java ${JBOSS_KIE_ARGS} ${stateFileInit}'"
-        java ${JBOSS_KIE_ARGS} $(getKieJavaArgs) ${stateFileInit}
+        # Workaround for RHPAM-4849
+        if [ -x "$(command -v java)" ]; then
```

Comment:
don't need to add this if imho, just use $JAVA_HOME/bin/java everywhere.

Reply:
Agree it would be better to use JAVA_HOME originally, but I choose this option because I did not want to change the original symlink in these scripts and also the warning might help users in cases when they using custom scripts in image with java command and facing some issues. 
Also the bug is cause probably by the image build process and not by these scripts.

Reply:
using java_home would not break anything and it is more consistent :)